README.md

```diff
@@ -30,40 +55,67 @@ It is also possible to select only certain fields as result, using the `--output
 znoyder packages --component network --tag osp-17.0 --output osp-patches
 ```
 
-There exist the `--debug` option which will display raw dictionary as output.
+There exists the `--debug` option which will display raw dictionary as output.
 It can be useful to find fields (key names) for `--output` option.
 
 ```
 znoyder releases --debug
 ```
 
-For further details, the standard `--help` shall guide you.
+
+## Descarger
+
+This tool downloads Zuul configuration, if there is any, from a given
+repository and its branch.
 
 ```
-znoyder --help
+descarger -r https://opendev.org/openstack/nova -b master -d zuul-config-files/
 ```
 
-# Usage examples - shperer
-After installing, the `shperer` command is available to use.
 
-Before running `shperer` CLI a set of projects should be available in the
+## Shperer
+
+It allows investigating details of Zuul configuration.
+
+Before running `shperer` CLI, a set of projects should be available in the
 local path.
 
-To get list of jobs and templates defined for a particular project for a
-particular zuul trigger simply type
+To get list of jobs and templates defined for a particular project
+and a Zuul trigger types, simply execute:
 
 ```
 shperer -d /path/to/neutron/ -b /path/to/templates/openstack-zuul-jobs -t check,gate
 ```
 
-There exist the `-v` verbose option.
+Note the tool requires absolute paths as an argument.
 
-For further details, the standard `--help` shall guide you.
+There exists the `-v` option for getting a verbose output with many details.
+
+
+## Fertiger
+
+It is mainly the templating tool for producing the adjusted configuration
+files for OSP-specifics. Calling it will produce a list of jobs considered
+during templates rendering and their remapping (if defined).
 
 ```
-shperer --help
+fertiger
```

Comment:
Could be nice if it supported JSON format

Reply:
Can be done – I am adding TODO item, thanks.